app/src/main/java/com/plaid/linksample/MainActivityStartActivityForResultJava.java

```diff
@@ -13,56 +13,47 @@
 import android.view.View;
```

Comment:
This is the previous `MainActivityJava`,  without the options menu to switch to other examples.